### PR TITLE
Update telegram and wordfeud app

### PR DESCRIPTION
**Pull request by janbosch**

### CODE REVIEW

**Comment by yonabuild** [comment]
Can one of the admins verify this patch?

File: dbinit/data/productionActivityCategories.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
          "com.skype.raider",
          "com.facebook.orca",
          "com.snapchat.android"
```
Comment (Bert-R) [inline dbinit/data/productionActivityCategories.json]: Missing trailing comma

File: dbinit/data/productionActivityCategories.json [hunk 2]
```diff
@@ -99,6 +100,7 @@
          "com.rovio.popcorn",
          "com.rovio.Toons.tv",
          "com.nianticlabs.pokemongo"
```
Comment (Bert-R) [inline dbinit/data/productionActivityCategories.json]: Same here.

**Comment by Bert-R** [comment]
add to whitelist

**Comment by Bert-R** [comment]
Looks good. I'm going to merge it.

**Pull request merged**